scripts/create_custom_video_dataset.py

```diff
@@ -0,0 +1,105 @@
+import argparse
```

Comment:
Given that this script is intended to be run via the CLI we could add the traditional "hashbang" line here and make it executable
```suggestion
#!/usr/bin/env python3
import argparse
```